pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -386,3 +386,55 @@ func createDivModNPackedDivmodV1Hinter(resolver hintReferenceResolver) (hinter.H
 
 	return newDivModNPackedDivmodV1Hint(a, b), nil
 }
+
+// IsZeroPack hint caculates packed value modulo SECP_P prime
+//
+// `newIsZeroPackHint` takes 1 operander as argument
+//   - `x` is the packed value that will be taken modulo SECP_P prime
+//
+// `newIsZeroPackHint` assigns the result as `x` in the current scope
+func newIsZeroPackHint(x hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "IsZeroPack",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
+			//> x = pack(ids.x, PRIME) % SECP_P
+
+			xAddr, err := x.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			xValues, err := vm.Memory.ResolveAsBigInt3(xAddr)
+			if err != nil {
+				return err
+			}
+
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
```

Comment:
if you want to add the comments line by line when implementing, can you do it for all lines of the hint?  i.e., add `//> x = pack(ids.x, PRIME) % SECP_P`

Reply:
Hm.. It was just a design choice I borrowed from another hint impl. I guess I will just remove it. The current impl is intuitive enough.